pandas/core/internals/managers.py

```diff
@@ -1602,6 +1521,45 @@ def from_array(cls, array: ArrayLike, index: Index) -> SingleBlockManager:
         block = new_block(array, placement=slice(0, len(index)), ndim=1)
         return cls(block, index)
 
+    def __getstate__(self):
```

Comment:
need for compat? can you remove some of this?

Reply:
this is basically the version that is in BaseBlockManager now, moved down to SingleBlockManager bc it is no longer used by BlockManager.  i do hope to get rid of some of this before long, dont have a good way to do so immediately

Reply:
(in case it was unclear, i dont expect to get rid of this for this PR)